docs/interchain-security.md

```diff
@@ -40,61 +40,65 @@ The Cost of Corruption calculation doesn't account for the availability of any s
 
-Initially it is up to validators to evaluate whether participation in a child chain validator set is worth the extra work and risk it imposes. Subsequently it is up to a delegator to decide whether the choices of their validator are aligned with their own. However it may be the case that a delegator agrees with how a validator operates on the parent chain and wishes to stay delegated there, but disagrees with the choice to participate (or not) on a child chain.
+Initially it is up to validators to evaluate whether participation in a Consumer Chain validator set is worth the extra work and risk it imposes. Subsequently it is up to a delegator to decide whether the choices of their validator are aligned with their own. However it may be the case that a delegator agrees with how a validator operates on the Provider Chain and wishes to stay delegated there, but disagrees with the choice to participate (or not) on a Consumer Chain.
 
-Initially the cost to validators to operate a new node instance on a new network may prohibit smaller validators from participating. It could be imagined that validating is a business of scale and the larger the operation the easier it is to scale further. This may result in only the largest validators from participating in Interchain Security. In order to ensure that delegators don't all redelegate their ATOMs to these super validators, it's important to build out the ability to delegate to validators on a per-chain basis. This would allow a delegator to delegate ATOMs to a small but well run validator on the Hub while also delegating the same ATOMs to a larger and less risk averse validator for a child chain at the same time. Should the larger validator misbehave and be slashed on the parent chain, the stake of that specific delegator would also be slashed. This would decrease the amount of stake the original small validator has delegated, but only with regard to that one delegator.
+Initially the cost to validators to operate a new node instance on a new network may prohibit smaller validators from participating. It could be imagined that validating is a business of scale and the larger the operation the easier it is to scale further. This may result in only the largest validators from participating in Interchain Security. In order to ensure that delegators don't all redelegate their ATOMs to these super validators, it's important to build out the ability to delegate to validators on a per-chain basis. This would allow a delegator to delegate ATOMs to a small but well run validator on the Hub while also delegating the same ATOMs to a larger and less risk averse validator for a Consumer Chain at the same time. Should the larger validator misbehave and be slashed on the Provider Chain, the stake of that specific delegator would also be slashed. This would decrease the amount of stake the original small validator has delegated, but only with regard to that one delegator.
```

Comment:
```suggestion
```

> Initially the cost to validators to operate a new node instance on a new network may prohibit smaller validators from participating.

So many small validators run all kinds of stuff. Big validators will not be able to just run everything that comes along because the software poses a risk to their systems. An ecosystem naturally will develop where small validators take bigger risks starting out to get established.

This whole Chain-Specific Delegations section needs to be taken out of this paper for now IMO.  It's a second-generation feature. It's very hard to understand for someone who is not familiar with interchain security (or steeped in the Cosmos core dev bubble) and it is not on our near-term roadmap.